awx/api/urls.py

```diff
@@ -3,15 +3,26 @@
 
 # noqa
 
-from django.conf.urls import include, patterns, url as original_url
+import six
+from django.conf.urls import include, url as original_url
+from importlib import import_module
 
-def url(regex, view, kwargs=None, name=None, prefix=''):
-    # Set default name from view name (if a string).
-    if isinstance(view, basestring) and name is None:
-        name = view
-    return original_url(regex, view, kwargs, name, prefix)
 
-organization_urls = patterns('awx.api.views',
+def scoped_url(module):
```

Comment:
I've always been on the fence about whether or not I like this translation mechanism rather than just referencing the view directly ala `OrganizationList.as_view()`.

This was one of the things that confused me when coming into Tower development a long time ago.

Do you think there's any benefit to carrying this logic vs dropping it in favor of the preferred method? It seems like we'd actually end up paying a not-insignificant price to call `import_module` + `getattr` for each view listed in this file for little benefit.

Reply:
I think this change continues with the current logic in a simple, compatible way. I wouldn't be opposed to changing the way we include views in `urls.py`, but it seems like it would add many more changes to an already large-ish PR and could wait.